code/modules/clothing/suits/armor.dm

```diff
@@ -244,6 +255,28 @@
 	)
 	slowdown = LIGHT_SLOWDOWN
 
+/obj/item/clothing/suit/armor/excelshock
+	name = "Excelsior Shock Trooper Armor"
+	desc = "Fresh from the design labs of Haven, the Excelsior Mk.1 Shock trooper armor combines speed and protection for its planetary assaults."
+	icon_state = "shock_ex"
```

Comment:
Please do not shorthand 'excelsior' to 'ex', use the full "shock_excelsior" to avoid any confusion people might have